src/main/resources/META-INF/resources/index.html

```diff
@@ -149,13 +162,13 @@
           input.attr('disabled', true);
           submitButton.attr('disabled', true).text('Analyzing...');
 
-          const args = { url: u.toString() };
+          const args = { urls: u.toString().split(',') };
```

Comment:
Does it need to trim the strings if you use a space after commas `", "`?

Reply:
It's not enforcing, maybe if it used a multiline box it could just split by line. I don't know which is easier/less error prone.

Reply:
pushed changes! :) thanks for the comment